### PR TITLE
test: increase test coverage

### PR DESCRIPTION
**Pull request by phm07**

### CODE REVIEW

File: internal/cmd/network/disable_protection_test.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package network_test
+
+import (
+	_ "embed"
```
Comment (jooola) [inline internal/cmd/network/disable_protection_test.go:4]: I see that the embed lib is sometimes imported but not used in the file, is that normal?

Comment (apricote) [reply internal/cmd/network/disable_protection_test.go:4]: That is required for `//go:embed` to work: https://pkg.go.dev/embed

Comment (jooola) [reply internal/cmd/network/disable_protection_test.go:4]: I understand, but we don't embed any file in this particular file.

Comment (apricote) [reply internal/cmd/network/disable_protection_test.go]: Removed all embed imports in files without actual embeds.

**Review by jooola — APPROVED** [review]

**Pull request merged**